mapadroid/worker/strategy/AbstractWorkerStrategy.py

```diff
@@ -410,7 +411,7 @@ async def _restart_pogo_safe(self):
         await asyncio.sleep(1)
         return await self.start_pogo()
 
-    async def _switch_user(self):
+    async def _switch_user(self, reason=None):
```

Comment:
Where is that reason used? What type should it have? Apparently string? Could use a type (imho enum) at this point